examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -187,10 +183,6 @@ def pad_fn(inputs):
     bounding_box_format="xywh", confidence_threshold=0.5, from_logits=True
 )
 
-for layer in model.feature_extractor.layers:
```

Comment:
Another option here would be to add a `--freeze_batch_norm` flag

Reply:
This is verifiably worse, I've ruin it like 10-15 times with and without randaugment, etc